yarn-project/end-to-end/src/e2e_nested_contract.test.ts

```diff
@@ -114,7 +114,9 @@ describe('e2e_nested_contract', () => {
       ];
 
       const tx = await new BatchCall(wallet, actions).send().wait();
-      const logs = L2BlockL2Logs.unrollLogs(await wallet.getUnencryptedLogs(tx.blockNumber!, 1)).map(toBigIntBE);
+      const logs = L2BlockL2Logs.unrollLogs(await wallet.getUnencryptedLogs(tx.blockNumber!, 1)).map(log =>
```

Comment:
I will make this nicer in [this PR](https://github.com/AztecProtocol/aztec-packages/pull/2600).